File: 2024/java/softeer/근무_시간.java
package softeer;
import java.io.*;
import java.util.*;
public class 근무_시간 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int ret = 0;
        for (int i = 0; i < 5; i++){
            String start = sc.next();
            String end = sc.next();
            int startH = Integer.parseInt(start.substring(0, 2));
            int startM = Integer.parseInt(start.substring(3, 5));
            int endH = Integer.parseInt(end.substring(0, 2));
            int endM = Integer.parseInt(end.substring(3, 5));
            ret += (60 - startM) + (endH - startH - 1) * 60 + endM;
        }
        System.out.println(ret);
        sc.close();
    }
}

File: 2024/python/softeer/근무시간.py
import sys

ret = 0

for _ in range(5):
    start, end = input().split()
    start_h = int(start[:2])
    start_m = int(start[3:5])
    end_h = int(end[:2])
    end_m = int(end[3:5])

    ret += (end_h - start_h) * 60
    ret += end_m - start_m

print(ret)